contracts/strategies/StrategyTUSDypool.sol

```diff
@@ -0,0 +1,494 @@
+/**
+ *Submitted for verification at Etherscan.io on 2020-11-16
+*/
+
+pragma solidity ^0.5.17;
+
+interface IERC20 {
```

Comment:
Requested access to origin repo to update the interfaces to the ones used on this repo. Otherwise feel free to apply the changes from the gist below so we can see them on this PR @orbxball 

https://gist.github.com/dmolina79/3a5232fb715916f71771f277d435bac5